src/main/java/Duke.java

```diff
@@ -1,10 +1,64 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
+
+        String line = "____________________________________________________\n";
```

Comment:
Constant should be named by CAPITAL LETTER for example String DIVIDER, String LOGO